bundles/org.openhab.binding.amberelectric/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -55,12 +55,20 @@
 		<label>Current Electricity Price</label>
 		<description>Current price to import power from the grid</description>
 		<category>Price</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Current</tag>
+		</tags>
 		<state readOnly="true" pattern="%.3f %unit%"/>
 	</channel-type>
 	<channel-type id="electricity-status">
 		<item-type>String</item-type>
 		<label>Current Electricity Status</label>
 		<description>Current price status of grid import</description>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
Suggest Status / Energy - as this is a Red / Yellow / Green indicator of if the power price is High / Medium / Low.